apps/qsim_amplitudes.cc

```diff
@@ -172,9 +178,9 @@ int main(int argc, char* argv[]) {
     }
   };
 
-  using Runner = QSimRunner<IO, BasicGateFuser<IO, GateQSim<float>>, Simulator>;
-
   Runner::Parameter param;
+  param.max_fused_size = opt.max_fused_size;
+  param.fuser_verbosity = opt.verbosity;
   param.seed = opt.seed;
   param.num_threads = opt.num_threads;
   param.verbosity = opt.verbosity;
```

Comment:
Is `param.verbosity` still in use separate from `param.fuser_verbosity`, or can it be removed?

Similarly in the other binary files.

Reply:
I removed `param.fuser_verbosity`.